test/db/cmd/cmd_i

```diff
@@ -1040,114 +1040,114 @@ CMDS=ic*
 EXPECT=<<EOF
 fs classes
 "f class.JNIEnv = 0x6060"
-"f method._JNIEnv.FindClass(char_const*) = 0x6060"
-"f method._JNIEnv.DeleteLocalRef(_jobject*) = 0x606a"
-"f method._JNIEnv.NewObject(_jclass*,__jmethodID*,_...) = 0x6074"
-"f method._JNIEnv.GetObjectClass(_jobject*) = 0x608e"
-"f method._JNIEnv.GetMethodID(_jclass*,_char_const*,_char_const*) = 0x6098"
-"f method._JNIEnv.CallObjectMethod(_jobject*,__jmethodID*,_...) = 0x60a4"
-"f method._JNIEnv.CallVoidMethod(_jobject*,__jmethodID*,_...) = 0x60c0"
-"f method._JNIEnv.GetStaticMethodID(_jclass*,_char_const*,_char_const*) = 0x60dc"
-"f method._JNIEnv.CallStaticObjectMethod(_jclass*,__jmethodID*,_...) = 0x60ea"
-"f method._JNIEnv.CallStaticIntMethod(_jclass*,__jmethodID*,_...) = 0x695e"
-"f method._JNIEnv.CallStaticVoidMethod(_jclass*,__jmethodID*,_...) = 0x697c"
+"f method._JNIEnv.FindClass_char_const = 0x6060"
```

Comment:
These differences are because now `ic*` and the setting code for method-flags share the same code to build the flag name. Before there were some discrepancies with how the names were sanitized, so this is more correct.